cvat-core/package.json

```diff
@@ -41,7 +41,7 @@
     "form-data": "^2.5.0",
     "jest-config": "^26.6.3",
     "js-cookie": "^2.2.0",
-    "jsonpath": "^1.1.0",
+    "json-logic-js": "^2.0.0",
```

Comment:
Why so many changes in package-lock if you only installed one package?

Reply:
I don't do nothing special. Just npm i. Is it important to store auto generated file in particular structure, while generation of lock files not guaranteed same structure through different iteration?

Reply:
@bsekachev , It looks like the number of changes are reasonable now.